packages/components/src/base-control/README.md

```diff
@@ -50,8 +51,7 @@ If this property is added, a help text will be generated using help property as
 
 ### className
 
-The class that will be added with "components-base-control" to the classes of the wrapper div.
-If no className is passed only components-base-control is used.
```

Comment:
Removed mentions of hardcoded class names.

Reply:
Good catch!